index.html

```diff
@@ -0,0 +1,136 @@
+        <li class="nav-item"><a class="nav-link" href="#home">Home</a></li>
+        <li class="nav-item"><a class="nav-link" href="./about.html">About</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Contact</a></li>
+        <li class="nav-item"><a class="nav-link" href="">Register</a></li>
+        <li class="nav-item convention-link"><a class="foursq-btn" href="#"><button>4sq convetion</button></a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+    
+  </header>
+  <section class="home">
+  
+    <div class="container">
+      <p class="theme">"Supernatural Encounter"</p>
+      <h1 class="heading">FOURSQUARE ANNUAL <br> CONVENTION 2023 <br> NAMIBIA</span></h1>
+      <h3 class="sub-heading">Why you should join us?</h3>
+      <p class="description">This year’s convention is very special in many ways, as we are working to ensure 
+        everybody enjoys the best camping experiences in the area of spiritual encounters, accommodation, 
+        transportation, security, water system, electricity, welfare, program spacing and hubs transmission system. 
+        The God of Supernatural Encounter has been faithful in helping us on all the projects we prayerfully 
+        embarked upon. If I were you, I would be there.
+      </p>
+      <div class="timestamp">
+        <h3>2023.05.17(WED) ~ 21(SUN)</h3>
+        <p>@Foursquare Gospel Church, Ongwediva, Namibia</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="program">
+
+    <div class="set-margin">
+      <div class="d-flex">
+        <h4>Main Program</h4>
+      </div>
+      <div class="cards">
+        <div class="card">
+          <i class="fa-solid fa-volume-high fa-3x"></i>
+          <h3>Praise & worship</h3>
+          <p>Time to praise and worship our living God.</p>
+        </div>
+        <div class="card">
+          <i class="fa-solid fa-people-roof fa-3x"></i>
+          <h3>Prayers</h3>
+          <p>Time to surrender all in the hands of God.</p>
+        </div>
+        <div class="card">
+          <i class="fa-solid fa-share-from-square fa-3x"></i>
+          <h3>Testimonies</h3>
+          <p>Time to share what the Lord has done for you or your family.</p>
+        </div>
+        <div class="card">
+          <i class="fa-solid fa-people-roof fa-3x"></i>
+          <h3>Sermon</h3>
+          <p>Time to hear what God has prepared for you.</p>
+        </div>
+      </div>
+      <div class="d-flex-btn d-flex">
+        <button class="btn text-2xl text-bold">Join our annual convention 2023</button>
+        <!-- button for desktop version -->
+        <button class="desktop-btn text-2xl text-bold">SEE THE WHOLE PROGRAM</button>
+      </div>
+    </div>
+  </section>
+
+  <section class="speakers">
+    <h2 class="featured-speakers">Featured Speakers</h2>
+    <hr class="hr1">
+    <ul class="speakers-list">
```

Comment:
- [ ] I kindly suggest that you add two more featured speaker to ensure it matches the template design given by having up to 6 featured speakers.


| ------------- | ------------- |
| ------------- | ------------- |
| Your Design | <img width="947" alt="image" src="https://user-images.githubusercontent.com/52934128/217908389-43eb621b-9ce8-4e19-a212-85e9d93d9a44.png">  |
| Template Design  | <img width="626" alt="image" src="https://user-images.githubusercontent.com/52934128/217908077-05ddb89a-9153-441b-9432-6a124b6520e7.png">  |